eo/notebook_config.yml

```diff
@@ -0,0 +1,9 @@
+# Config file containing the list of notebooks to download for a specific image.
+#
+# Used in the deploy-data-specific-image script from birdhouse-deploy.
+# More details on this config can be found on this script file :
+#   https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/deployment/deploy-data-specific-image
+
+- repo_url: https://github.com/crim-ca/pavics-jupyter-images
+  branch: master
```

Comment:
To be really safe, use `origin/master` because `master` is the local branch ... which might not always track the proper `origin/master`.

Same comment for the other `nlp/notebook_config.yml` as well.

Reply:
I think `master` is okay here. The way it works in the script is that it builds a full url using the values in the config here. 
In this code `FULL_URL=$GIT_REPO/branches/$BRANCH/$SOURCE_PATH`, the branch parameter is just inserted in the url to access to the right place on the online repo. No need to distinguish between local or origin since it is a path found on the github site.

Reply:
Oh yes indeed you are right, my bad.  You do not perform a checkout so there is no local vs remote flavor of `master` to worry about.  I was thinking in term of my script which does a local checkout.  Sorry for the false positive.

Reply:
No problem, I understand where it was coming from! :)